install.sh

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2020, The Pallene Developers
+# Pallene is licensed under the MIT license.
+# Please refer to the LICENSE and AUTHORS files for details
+# SPDX-License-Identifier: MIT
+
+####
+# One script to rule them all, one script to find them,
+# one script to bring them all and in the ease-of-use bind them.
+####
+
+# Versions
+LUA_VERSION=5.4.7
+LUAROCKS_VERSION=3.9.1
```

Comment:
Unlike the LUA_VERSION and PT_VERSION, the LUAROCKS_VERSION is only there because our CI needs a Lua version. In theory we could always go with a more recent version. (And in fact, it's probably about time we bump this to a newer Luarocks...)